### DISCUSSION TITLE
Require branch up to date without status check

### DISCUSSION THREAD

**Discussion by SampsonCrowley** [community/community#20606] [Pull Requests]
Is there a good reason that requiring a branch to be up to date is dependent on status checks?

It makes absolutely zero sense to me that you should be required to have *additional* status checks if you do not want people to be able to merge stale branches
![Screen Shot 2022-07-15 at 8 28 54 AM](https://user-images.githubusercontent.com/22461638/179260487-9ac1e651-1706-424b-af03-a0bc72c27203.png)
Why is this dependent on the other? why is requiring a branch to be up to date not it's own independent rule?

**Comment by at-ng** [comment]
Related discussion https://github.com/community/community/discussions/8940
